ember-cli-build.js

```diff
@@ -1,8 +1,8 @@
 /* eslint-env node */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
+const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
 
 module.exports = function (defaults) {
-  var app = new EmberAddon(defaults, {
+  const app = new EmberAddon(defaults, {
```

Comment:
Isn't that only because they officially support old node versions, whereas we don't? Why do we want `var` anywhere?